pkg/interactive/interactive_client.go

```diff
@@ -669,3 +677,7 @@ func (c *InteractiveClient) startCancelHandler() chan bool {
 	}()
 	return quitChannel
 }
+
+func (c *InteractiveClient) enabeAutoComplete(buffer *prompt.Buffer) {
```

Comment:
It doesn't look like this function is getting called from anywhere. Does this still need to be here? You may remove it if not!